src/app/beer_garden/api/stomp/server.py

```diff
@@ -0,0 +1,136 @@
+import stomp
+import logging
+import time
+import beer_garden.config as config
+from brewtils.schema_parser import SchemaParser
+import beer_garden.events
+import beer_garden.router
+from beer_garden.api.stomp.processors import append_headers, process_send_message
+
+conn = None
+bg_active = False
+
+
+def send_message(message, headers=None):
+    if headers is None:
+        headers = {}
+    global conn
+    stomp_config = config.get("entry.stomp")
+    message, response_headers = process_send_message(message)
+    response_headers = append_headers(
+        response_headers=response_headers, request_headers=headers
+    )
+    if conn.is_connected():
+        if "reply-to" in headers:
+            conn.send(
+                body=message, headers=response_headers, destination=headers["reply-to"]
+            )
+        else:
+            conn.send(
+                body=message,
+                headers=response_headers,
+                destination=stomp_config.event_destination,
+            )
+
+
+def send_error_msg(error_msg, headers):
+    global conn
+    stomp_config = config.get("entry.stomp")
+    error_headers = None
+    error_headers = append_headers(error_headers, headers)
+
+    if conn.is_connected():
+        if "reply-to" in headers:
+            conn.send(
+                body=error_msg,
+                headers=error_headers,
+                destination=headers["reply-to"],
+            )
+        else:
+            conn.send(
+                body=error_msg,
+                headers=error_headers,
+                destination=stomp_config.event_destination,
+            )
+    pass
+
+
+class OperationListener(stomp.ConnectionListener):
+    def on_error(self, headers, message):
+        logger = logging.getLogger(__name__)
```

Comment:
This logger should probably be a module-level thing (on the same level as `conn` and `bg_active`).